{{ cookiecutter.package_name }}/.pre-commit-config.yaml

```diff
@@ -20,6 +25,13 @@ repos:
         exclude: ".*(.fits|.fts|.fit|.header|.txt|tca.*|.json)$|^CITATION.rst$"
       - id: mixed-line-ending
         exclude: ".*(.fits|.fts|.fit|.header|.txt|tca.*)$"
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.6
+    hooks:
+      - id: codespell
+        additional_dependencies:
+          - tomli
```

Comment:
As our config is no longer in pyproject.toml presumably this isn't needed? @nabobalis 
```suggestion
```